src/aap_eda/services/project/imports.py

```diff
@@ -56,8 +56,10 @@ def _project_import_wrapper(
 ):
     @wraps(func)
     def wrapper(self: ProjectImportService, project: models.Project):
-        project.import_state = models.Project.ImportState.RUNNING
-        project.save(update_fields=["import_state"])
+        # The view already changed the state to running before importing
```

Comment:
Where this happen? The view set the status to pending as far as I can see. And I think that's correct. We should not set the running before the import task begins.

Reply:
sorry. I misread the code.